integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonCreateTableCommand.scala

```diff
@@ -110,7 +111,8 @@ case class CarbonCreateTableCommand(
                |  tableName "$tableName",
                |  dbName "$dbName",
                |  tablePath "$tablePath",
-               |  path "$tablePath"
+               |  path "$tablePath",
+               |  isCarbonTableVisibility "$isCarbonTableVisibility"
```

Comment:
1. Rename isCarbonTableVisibility to isCarbonTableVisibile
2. Add a comment to specify in which case this property will be used and what is the benefit of using this property